src/components/DefaultAlert/style.ts

```diff
@@ -0,0 +1,10 @@
+import { Alert } from '@mui/material';
+import styled from '@emotion/styled';
+
+export const CustomAlert = styled(Alert)`
+  position: fixed;
+  top: 10%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 1000000;
+`;
```

Comment:
![image](https://user-images.githubusercontent.com/66072832/183281904-7127c060-ddce-47ca-828b-37f34a0a9b94.png) ![image](https://user-images.githubusercontent.com/66072832/183281910-1658d507-17bb-4f12-bd47-89832a919c2d.png)


```suggestion
   color: #fff;
`;
```

success, warning에는 color가 error, information 처럼 흰색으로 지정되지 않는데, CustomAlert에 color도 지정해주면 좋을 것 같습니다!